tests/feature_point_id/test_feature_point_id.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import cv2 as cv
+from unittest.mock import  patch

```

Comment:
```suggestion
from unittest.mock import patch
```
静的解析ツールはこういうのも発見できる？

Reply:
自動で発見されるから、コメントする必要がなくなる！